VAMobile/src/store/api/types/Errors.ts

```diff
@@ -79,43 +78,33 @@ export const DowntimeFeatureTypeConstants: {
   rx: 'rx_refill',
 }
 
-// Friendly names shown in downtime messages
-export const DowntimeFeatureNameConstants = {
-  [DowntimeFeatureTypeConstants.facilityLocator]: 'Facility Locator',
-  [DowntimeFeatureTypeConstants.claims]: 'Claims',
-  [DowntimeFeatureTypeConstants.appointments]: 'Appointments',
-  [DowntimeFeatureTypeConstants.secureMessaging]: 'Secure Messaging',
-  [DowntimeFeatureTypeConstants.letters]: 'Letters',
-  [DowntimeFeatureTypeConstants.directDepositBenefits]: 'Direct Deposit',
-  [DowntimeFeatureTypeConstants.disabilityRating]: 'Disability Rating',
-  [DowntimeFeatureTypeConstants.militaryServiceHistory]: 'Military Service History',
-  [DowntimeFeatureTypeConstants.appeals]: 'Appeals',
-  [DowntimeFeatureTypeConstants.userProfileUpdate]: 'Personal Information',
-  [DowntimeFeatureTypeConstants.payments]: 'Payments',
-  [DowntimeFeatureTypeConstants.rx]: 'VA Prescriptions',
+export const ScreenIDToFeatureName = {
```

Comment:
I've changed this to be more screen driven and less feature driven. To me, it feels more natural to combine pieces of data to make the relationships clearer, like:
```
screenId: {
  name: 'what i want the user to see',
  features: ['claims', 'appeals', 'whatever']
}
```

But I'm new to typescript and chose not to change too much. Let me know if anyone else would find that clearer and I could give it a try.